README.md

```diff
@@ -0,0 +1,191 @@
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+npm installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone git@github.com:Oscar-Espinoza/Calculator.git
+  cd Calculator
+-
+
+### Install
+
+Install this project with:
+
+
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+npm run test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@Oscar-Espinoza](https://github.com/githubhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Scientific Functions]**
+- [ ] **[Unit Conversion]**
+- [ ] **[Currency Conversion]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Oscar-Espinoza/Calculator/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+I am excited to share my new calculator app with you. This app is designed to be a versatile and user-friendly tool for all your calculation needs. Whether you are a student, a professional, or simply someone who needs to do math on a daily basis, I believe this app will be a valuable asset for you.
+
+I have put a lot of time and effort into developing this app, and I hope you will find it useful. If you do, I would greatly appreciate your support. By downloading and using my app, you will be helping me to continue improving and updating it with new features and functionality.
+
+Thank you in advance for your support. I hope you enjoy using my app as much as I enjoyed creating it.
+
+Sincerely,
+
+[Oscar Espinoza]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank... -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
```

Comment:
- [ ] I think it is better to remove this section of your readme if you have nothing to specify under it